svir/dialogs/load_output_as_layer_dialog.py

```diff
@@ -301,6 +302,20 @@ def populate_rlz_or_stat_cbx(self):
         self.rlz_or_stat_cbx.setEnabled(True)
         self.rlz_or_stat_cbx.addItems(self.rlzs_or_stats)
 
+    def get_loss_types(self, session, hostname, calc_id):
+        composite_risk_model_attrs = extract_npz(
+            session, hostname, calc_id, 'composite_risk_model.attrs',
+            message_bar=self.iface.messageBar())
+        if composite_risk_model_attrs is None:
+            return
```

Comment:
Can this happen? When?

Reply:
It sounds strange to me too... I am going to investigate.